docs/FEATURES.md

```diff
@@ -1,32 +1,75 @@
+| Base add-ons                | Location types            | One **location_type** value in [stops.txt](https://gtfs.org/schedule/reference/#stopstxt)    | https://gtfs.org/getting_started/features/base_add-ons/#location-types |
+| Base add-ons                | Feed Information          | One line of data in [feed_info.txt](https://gtfs.org/schedule/reference/#feed_infotxt)  |  https://gtfs.org/getting_started/features/base_add-ons/#feed-information |
+| Base add-ons                | Attributions              | One line of data in [attributions.txt](https://gtfs.org/schedule/reference/#attributionstxt) |   https://gtfs.org/getting_started/features/base_add-ons/#attributions|
+| Base add-ons                  | Shapes                    | One line of data in [shapes.txt](https://gtfs.org/schedule/reference/#shapestxt)      |   https://gtfs.org/getting_started/features/base_add-ons/#shapes |   
+| Base add-ons               | Transfers                 | One line of data in [transfers.txt](https://gtfs.org/schedule/reference/#transferstxt)     | https://gtfs.org/getting_started/features/base_add-ons/#transfers  |
+| Base add-ons | Frequency-Based Service | One line of data in [frequencies.txt](https://gtfs.org/schedule/reference/#frequenciestxt)    | https://gtfs.org/getting_started/features/base_add-ons/#frequency-based-service  |
+| Fares                    | Fare Products             | One line of data in [fare_products.txt](https://gtfs.org/schedule/reference/#fare_productstxt)  | https://gtfs.org/getting_started/features/fares/#fare-products |                                                                                                                      
+| Fares                    | Fare Media                | One line of data in [fare_media.txt](https://gtfs.org/schedule/reference/#fare_mediatxt) |  https://gtfs.org/getting_started/features/fares/#fare-media |
+| Fares                    | Route-Based Fares         | One **network_id** value in [routes.txt](https://gtfs.org/schedule/reference/#routestxt)<br/>OR<br/>one line of data in [networks..txt](https://gtfs.org/schedule/reference/#networkstxt)   |    https://gtfs.org/getting_started/features/fares/#route-based-fares   |
+| Fares                    | Time-Based Fares          | One line of data in [timeframes.txt](https://gtfs.org/schedule/reference/#timeframestxt)   | https://gtfs.org/getting_started/features/fares/#time-based-fares  |
+| Fares                    | Zone-Based Fares          | One line of data in [areas.txt](https://gtfs.org/schedule/reference/#areastxt)   |     https://gtfs.org/getting_started/features/fares/#zone-based-fares |
+| Fares                    | Fares Transfers           | One line of data in [fare_transfer_rules.txt](https://gtfs.org/schedule/reference/#fare_transfer_rulestxt)    |    https://gtfs.org/getting_started/features/fares/#fares-transfers |
```

Comment:
@tzujenchanmbd and I had a discussion about this one and decided to change it to `Fare Transfers` (first word singular and second word plural). We'll update the documentation in GTFS.org shortly to reflect this. The name should also be updated in line 59 (in the Feature Migration List).